gateway-server/server.py

```diff
@@ -0,0 +1,91 @@
+from flask import Flask, request, jsonify, render_template
+import requests
+# TODO: from kazoo.client import KazooClient
+from pymongo import MongoClient
+import random
+
+app = Flask(__name__)
+
+# TODO: Add Zookeeper functionality
+
+mongo_uri = '<hidden-for-pull-request>'
```

Comment:
You can use a github secret for this: https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions